mongorestore/empty_timestamp.go

```diff
@@ -0,0 +1,152 @@
+package mongorestore
+
+import (
+	"bytes"
+	"slices"
+
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+var nineNuls = make([]byte, 9, 9)
+
+// FindZeroTimestamps returns all document paths (each represented as
+// a []string) that contain a zero-value timestamp.
+func FindZeroTimestamps(raw bson.Raw) ([][]string, error) {
+	if !bytesSuggestEmptyTimestamp(raw) {
+		return nil, nil
+	}
+
+	// TODO: log(trace, "found empty-timestamp byte sequence")
```

Comment:
Leftover? Why can't we trace this?

Reply:
This was a note-to-self that I wrote while writing this function, but I’m not sure it’s worth logging about TBH.